lib/html-templates.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/**
+ * Class HTMLTemplates
+ *
+ * Static methods to return HTML templates. Used in `templates/comments.php`
+ *
+ * Templates and implementation copied from @aliso's commit:
+ * https://github.com/10up/wp-discourse/commit/5c9d43c4333e136204d5a3b07192f4b368c3f518
+ */
+class HTMLTemplates {
+
+  /**
+   * HTML template for replies
+   *
+   * Can be customized from within a theme using the filter provided.
+   *
+   * Available tags:
+   * {comments}, {discourse_url}, {discourse_url_name},
+   * {topic_url}, {more_replies}, {participants}
+   *
+   * @static
+   * @return mixed|void
+   */
+  public static function replies_html() {
+    ob_start();
+    ?>
+    <div id="comments" class="comments-area">
+      <h2 class="comments-title">Notable Replies</h2>
+      <ol class="comment-list">{comments}</ol>
+      <div class="respond comment-respond">
+        <h3 id="reply-title" class="comment-reply-title">
+          <a href="{topic_url}">Continue the discussion at</a> at {discourse_url_name}
+        </h3>
+        <p class="more-replies">{more_replies}</p>
+        <p class="comment-reply-title">{participants}</p>
+      </div><!-- #respond -->
+    </div>
+    <?php
+    return apply_filters( 'discourse_replies_html', ob_get_clean() );
```

Comment:
I'd say it's better to have `ob_get_clean` called outside the `apply_filters` (here and everywhere else).

My reasoning is this: if you want only to append some extra markup, you'll have to remember to clean the buffer on return. I think it's an unnecessary extra step...